src/Qir/Execution/CommandLineTool/Microsoft.Quantum.Qir.CommandLineTool.csproj

```diff
@@ -3,7 +3,13 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
     <PackAsTool>true</PackAsTool>
+    <ToolCommandName>QirCommandLineTool</ToolCommandName>
```

Comment:
This is the actual command you type to invoke the command from the shell.
`QirCommandLineTool` is not something that I would like to keep typing from the command line.
What about just `qir` or `qir-cli`?

Reply:
Since this is something that is very visible to those that would use the tool, I'd like to get other's input on this. @bettinaheim @cesarzc @swernli What are other peoples thoughts on what the command should be?

Reply:
For my opinion, I like the shorter `qir-cli`. It also somewhat depends on if there is any plans on expanding the responsibilities for this tool in the future.

Reply:
Going with `qir-cli` for the tool name.